docs/design-docs/sync.md

```diff
@@ -13,8 +13,8 @@ WordPress Studio Sync enables developers to pull a live site down for local deve
 - **Sync**: Replicating files and database content between a local machine and a remote site in any direction.
 - **Push**: Copying changes from a local machine to a remote (staging or production) site.
 - **Pull**: Copying changes from a remote site down to a local machine.
-- **Staging site**: A staging site is hosted on WordPress.com and is connected to its production site. WordPress.com staging sites can sync with production sites and vice versa. However, that is a different feature managed entirely in the wp.com Hosting Features web interface.
-- **Production site**: A production site hosted on WordPress.com or any Pressable site. For Pressable sites, we currently cannot identify if a site is a production or staging and we don't display any tag or label for them.
+- **Staging site**: A staging site hosted on WordPress.com or a Pressable site with the environment type set to `staging`. WordPress.com staging sites can sync with production sites and vice versa. However, that is a different feature managed entirely in the wp.com Hosting Features web interface.
```

Comment:
```suggestion
- **Staging site**: A staging site hosted on WordPress.com or a Pressable site with the environment type set to `staging`. WordPress.com staging sites can sync with production sites and vice versa. However, that is a different feature managed entirely in the WordPress.com Hosting Features web interface.
```

Reply:
Thanks, Rahul! Good idea! I have just applied it.